.github/workflows/examine-wasm-bindings.yml

```diff
@@ -0,0 +1,234 @@
+name: Examine the Wasm Bindings
+on:
+  push:
+    branches:
+      - develop
+      - production
+      - bindings/wasm
+    paths:
+      - ".github/workflows/examine-wasm-bindings.yml"
+      - ".github/actions/**"
+      - "bindings/wasm/**"
+      # Watch the Rust core too
+      - "src/**"
+      - "Cargo.toml"
+      - "Cargo.lock"
+  pull_request:
+    branches:
+      - develop
+      - production
+    paths:
+      - ".github/workflows/examine-wasm-bindings.yml"
+      - ".github/actions/**"
+      - "bindings/wasm/**"
+      # Watch the Rust core too
+      - "src/**"
+      - "Cargo.toml"
+      - "Cargo.lock"
+  schedule:
+    - cron: "0 1 * * *"
+  workflow_dispatch:
+
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: Test
+    if: ${{ ! github.event.schedule }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest, macos-latest, ubuntu-latest]
+        node: ["16"]
+
+    steps:
+      - name: Checkout the Source Code
+        uses: actions/checkout@v3
+
+      - name: Set Up Nightly Rust
+        uses: ./.github/actions/setup-rust
+        with:
+          toolchain: nightly
+          target: wasm32-unknown-unknown
+          cache: true
+          cache-root: bindings/wasm/
+          cache-job-id: ${{ github.workflow }}-${{ github.job }}-${{ matrix.os }}-${{ matrix.node }}
+          cache-hash: ${{ hashFiles('.github/workflows/examine-wasm-bindings.yml') }}
```

Comment:
Is it necessary to create a new cache artifact because this file changed? The Cargo home and target cache artifacts are just under 400 MB (combined) for each OS

Reply:
No idea 🙈 I jsut copied https://github.com/iotaledger/iota.rs/blob/develop/.github/workflows/examine-wasm-bindings.yml

Reply:
I don't think it's necessary since it doesn't really indicate anything has changed dependency-wise, but maybe @Thoralf-M had a reason for doing that in `iota.rs`

Reply:
I just copied it from the identity repo :trollface: I guess it's fine to remove